pages/implementation/property-groups-unfolding.mdx

```diff
@@ -0,0 +1,40 @@
+---
+layout: 'guide'
+title: 'Start using Avo Visual Debugger in code'
+---
+
+import Link from '../../components/Link';
+
+In Avo you can group multiple event properties into a property group.
```

Comment:
I think it would be great to start with explaining the value of property groups. Perhaps something like:
```suggestion
Property Groups allow you to create a set of event properties that are commonly used together to increase the consistency of your tracking plan as well as increasing efficiency when defining new events and implementing them.
```

Reply:
Added